.devcontainer/devcontainer.json

```
@@ -0,0 +1,33 @@
+{
+    "image": "ghcr.io/citusdata/citus-devcontainer:dev-devcontainer",
```

Image tag needs to be updated right before merging to reflect that the image will be built from the `main` branch

```suggestion
    "image": "ghcr.io/citusdata/citus-devcontainer:main",
```